requirements.txt

```diff
@@ -1,10 +1,10 @@
-git+https://github.com/metabrainz/brainzutils-python.git@v1.4.1
+git+https://github.com/metabrainz/brainzutils-python.git@v1.5.0
 Fabric == 1.10.2
 Flask-Testing == 0.4.2
 Flask-SQLAlchemy==2.0
 Flask-UUID == 0.2
 Jinja2 == 2.8
-SQLAlchemy==1.0.8
+SQLAlchemy==1.2.5
```

Comment:
Should update this here too: https://github.com/metabrainz/messybrainz-server/blob/master/setup.py#L16

Reply:
done